docs/components/base/_index.md

```diff
@@ -521,6 +521,62 @@ myBaseTurningRadius := properties.TurningRadiusMeters
 {{% /tab %}}
 {{< /tabs >}}
 
+### GetGeometries
+
+Get all the geometries associated with the base in its current configuration, in the [frame](/services/frame-system/) of the base.
```

Comment:
Maybe give a pointer here that geometries are usually used by motion/navigation and are sometimes required if obstacle avoidance is wanted in your smart machine stack using our builtin services.